### PR TITLE
Feature implement persistence

### PR DESCRIPTION
**Pull request by yxyx-github**
Issue #65

### CODE REVIEW

File: src/main/java/hwr/oop/group4/todo/persistence/FileAdapter.java [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
         }
     }
 
     @Override
-    public Persistable load(File file) {
-        return null;
+    public Persistable load(Persistable data, File file) {
+        StringBuilder output = new StringBuilder();
+        try {
+            Scanner scanner = new Scanner(file);
```
Comment (termitehuegel) [inline src/main/java/hwr/oop/group4/todo/persistence/FileAdapter.java:29]: scanner should be closed => use try with resource

File: src/main/java/hwr/oop/group4/todo/persistence/PersistableTodoList.java [hunk 2]
```diff
@@ -15,4 +15,9 @@ public String exportAsString() {
         JSONObject todoListObject = new JSONObject(todoList);
         return todoListObject.toString();
     }
+
+    @Override
+    public void importFromString(String dataString) {
+
```
Comment (termitehuegel) [inline src/main/java/hwr/oop/group4/todo/persistence/PersistableTodoList.java:21]: fill with code or add a comment

**Review by termitehuegel — CHANGES_REQUESTED** [review]

**Comment by yxyx-github** [comment]
The second part (load from file and convert) still work in progress. I will let you know when I'm ready.

**Comment by yxyx-github** [comment]
~~The second part (load from file and convert) still work in progress. I will let you know when I'm ready.~~

I should have taken a different branch for additional stuff that do not address this issue.

**Review by termitehuegel — APPROVED** [review]
approved for consultation

**Pull request merged**